plugins/occt/module/vtkF3DOCCTReader.cxx

```diff
@@ -260,7 +270,10 @@ class vtkF3DOCCTReader::vtkInternals
     polydata->SetLines(linesCells);
 
 #if F3D_PLUGIN_OCCT_XCAF
-    polydata->GetCellData()->SetScalars(colors);
+    if (colors->GetSize() > 0)
```

Comment:
please add a comment stating that this may be empty if OCCT does not provide a ColorTool